### ISSUE TITLE
suspicious claim of parasite wasp (Trichogramma sibiricum) parasitizing a plant (Vaccinium macrocarpon)

### ISSUE THREAD

**Issue by jhpoelen** [globalbioticinteractions/natural-history-museum-london-interactions-bank#6]
in https://github.com/ropensci/rglobi/issues/50#issuecomment-1503164356 @pozsgaig writes:

> I found that querying some parasitic wasps also brings up plant species as being parasitised. These wasps are host specific and linked to Lepidoptera species, so this should not happen. When I searched the source, I found that, for example, Trichogramma sibiricum 'is parasite of' Vaccinium macrocarpon. This hit also comes up on the website, and I checked where does it originally come from. The record is from the NHMS's Universal Chalcidoidea Database (UCD) and for this particular wasp species that database lists several Lepidoptera as hosts and some plants (also bacteria) as well as ASSOCIATED SPECIES. Based on the linked article on the site (https://www.nhm.ac.uk/resources/research-curation/projects/chalcidoids/pdf_X/HaleEl2002.pdf), the wasps were used as biocontrol agents on cranberry (Vaccinium) plants against the Lepidoptera pest. Thus, it seems, that the interaction is erroneously recorded in GLOBI. If all interactions from UCD were automatically put down as parasitic relationships, this can be a rather large issue and impact thousands of records. I am, actually, not sure this has caused the discrepancies reported in this issue - but there is a fair chance.

possibly related to #5

**Comment by jhpoelen** [comment]
The following record was present in record provided by NHM London - 

```
InteractionID          
InteractionURL         
Species1UUID           Tricho sibirSa
Species1Name           Trichogramma sibiricum
Species1LifeCycleStage 
Species1OrganismPart   
Species1Status         
InteractionType        is parasite of
InteractionOntologyURL http://purl.obolibrary.org/obo/RO_0002444
Species2UUID           17461
Species2Name           Vaccinium macrocarpon
Species2LifeCycleStage 
Species2OrganismPart   
Species2Status         
LocationUUID           
LocationName           
LocationCountryName    
ISO2                   
Importance             
InteractionRecordType  
Reference              Pinto999 via Universal Chalcidoidea Database
ReferenceDOI           
Reference Page         200
Notes                  
```

as found via:

```
preston track "https://data.nhm.ac.uk/dataset/82e807f0-6273-4f19-be0a-7f7558442a25/resource/f84ad9f1-bd0c-4811-8108-8dcd83f224a4/download/nhminteractionsbank.csv"\
 | grep hasVersion\
 | preston cat\
 | cut -d $'\r' -f 1,339556\
 | mlr --icsv --oxtab
```

**Comment by jhpoelen** [comment]
So, it appears that the suspicious claim reported by @pozsgaig is mentioned in a data source. 

Perhaps @edwbaker can give us some insights into why the NMH claims that a parasite wasp (Trichogramma sibiricum) is parasitizing a plant (Vaccinium macrocarpon) . 

@edwbaker if there's another contact for the NMH Interaction Bank resource, please do share, and I'll redirect my query.

**Comment by jhpoelen** [comment]
I've attached a csv with all claims with Reference column value that match:

```regex
.*Universal Chalcidoidea Database.*
```

[NHM_London_Universal_Chalcidoidea_Database.csv.zip](https://github.com/globalbioticinteractions/natural-history-museum-london-interactions-bank/files/11200646/NHM_London_Universal_Chalcidoidea_Database.csv.zip)

generated via:

```
preston ls\
 | grep hasVersion\
 | preston cat\
 | tr '\r' '\n'\
 | mlr --csv filter '$Reference =~ ".*Universal Chalcidoidea Database.*"'\
 | tee NHM_London_Universal_Chalcidoidea_Database.csv
```

with ```preston head``` being ```hash://sha256/eee525f744a3e07fa6a71de9c7c3daebf6efad2373b0571170004a07bfb396a3``` as documented in attached 
[eee525f744a3e07fa6a71de9c7c3daebf6efad2373b0571170004a07bfb396a3.zip](https://github.com/globalbioticinteractions/natural-history-museum-london-interactions-bank/files/11200684/eee525f744a3e07fa6a71de9c7c3daebf6efad2373b0571170004a07bfb396a3.zip)
.

**Comment by jhpoelen** [comment]
@pozsgaig can you please take a peek at https://github.com/globalbioticinteractions/natural-history-museum-london-interactions-bank/files/11200646/NHM_London_Universal_Chalcidoidea_Database.csv.zip and confirm that the relations claims may have the incorrect interaction type (e.g., parasite of should be associated with)

**Comment by pozsgaig** [comment]
@jhpoelen Yes, many of these interaction types are incorrect, for example, everything between Trichogramma* and any plant species (all recorded as parasites). When you manually search the UCD though, these are correctly marked as 'associated with'.

**Comment by jhpoelen** [comment]
Please note that I've just communicated your observations with curators of NHM Interaction Bank at https://data.nhm.ac.uk/dataset/nhm-ib via some web form and by email to John Noyes, Vince Smith, Ed Baker and Deb Paul. Curious to see what comes of this . . .

**Comment by jhpoelen** [comment]
see related #7
